src/js/components/Accordion.js

```diff
@@ -1,6 +1,7 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
-import React, { Component, PropTypes } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Yes. It'd be quite easy given that React import is done in a consistent manner in Grommet. 

I'm travelling right now. Will spin up a script in some time and push.